supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -152,6 +166,23 @@
 
 *See also*:
 
+[discrete]
+[[privilege]]
+==== image:images/yes.png[yes] permission (noun)
+*Description*: A "privilege" is a right granted to an agent to perform actions only available to agents configured with that privilege. A user is one example of an agent.
```

Comment:
```suggestion
*Description*: A "privilege" is a right granted to an agent to perform certain restricted actions. A user is one example of an agent.
```

Reply:
@bburt-rh Rewrote to avoid reusing the defined term in the definition.

Reply:
fixed per your suggestion